libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -99,9 +106,9 @@ typedef enum MQTTStatus
  */
 typedef enum MQTTQoS
 {
-    MQTTQoS0 = 0, /**< Delivery at most once. */
-    MQTTQoS1 = 1, /**< Delivery at least once. */
-    MQTTQoS2 = 2  /**< Delivery exactly once. */
+    MQTTQoS0 = MQTT_QOS_0, /**< Delivery at most once. */
+    MQTTQoS1 = MQTT_QOS_1, /**< Delivery at least once. */
+    MQTTQoS2 = MQTT_QOS_2  /**< Delivery exactly once. */
```

Comment:
I really don't see the point in having a macro and enum for QoS, it's just confusing to me.